cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -368,3 +372,52 @@ def _clear_ftp():
         collect_logs.collect_current()
 
     check_ftp(appliance, log_depot.ftp, appliance.server.name, appliance.server.sid)
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1452224])
+def test_log_collection_over_ipv6(log_depot, depot_machine_ipv4_and_ipv6, appliance, request):
+    """
+    Bug 1452224 - Log Collection fails via IPv6
+
+    Bugzilla:
+        1452224
+
+    An IPv6 FTP server can be validated for log collection, and log
+    collection succeeds.
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+    """
+    ipv4, ipv6 = depot_machine_ipv4_and_ipv6
+    # FTPClient gives error with ipv6
+    log_depot.machine_ip = ipv4
+
+    @request.addfinalizer
+    def _clear_ftp():
+        with log_depot.ftp as ftp:
+            ftp.cwd(ftp.upload_dir)
+            ftp.recursively_delete()
+
+    # Prepare empty workspace
+    with log_depot.ftp as ftp:
+        # move to upload folder
+        ftp.cwd(ftp.upload_dir)
+        # delete all files
+        ftp.recursively_delete()
+
+    uri = "{ipv6}{log_depot.access_dir}"
```

Comment:
Missing the `f` here before the string, pretty sure you don't want this raw string being used for the URI.